catalog/models.py

```diff
@@ -142,25 +142,25 @@ def tagged(self, tags):
         return self.get_queryset().tagged(tags)
 
 
-# @todo #257:30m  Document terms Product, Position and Option
-#  Seems the best way is to do it with subclassing.
-#  Such documentation will clarify types and will be obvious for programmers.
-#  But check the docs too. May be some of this terms are mentioned there.
-class AbstractProduct(models.Model, AdminTreeDisplayMixin):
+# @todo #261:60m  Use the new catalog arch at sites.
+class Position(models.Model):
```

Comment:
Let's add here `Abstract` prefix or drop it from other abstract models

Reply:
i'd prefer to add it